doc/plutus-core-spec/untyped-cek-machine.tex

```diff
@@ -53,9 +54,12 @@ \section{The CEK Machine}
 \begin{figure}[H]
     \centering
     \[\begin{array}{lrclr}
-        \textrm{Frame} & f  & ::=   & \inForceFrame              & \textrm{force}\\
-                       &    &       & \inAppLeftFrame{(M,\rho)}  & \textrm{left application}\\
-                       &    &       & \inAppRightFrame{V}        & \textrm{right application}
+        \textrm{Frame} & f  & ::=   & \inForceFrame                                                       & \textrm{force}\\
+                       &    &       & \inAppLeftFrame{(M,\rho)}                                           & \textrm{left application to term}\\
+                       &    &       & \inAppLeftFrame{V}                                                  & \textrm{left application to value}\\
+                       &    &       & \inAppRightFrame{V}                                                 & \textrm{right application of value}\\
+                       &    &       & \inConstrFrame{i}{(\repetition{M}, \rho)}{\repetition{V}}           & \textrm{constructor argument}\\
```

Comment:
Shouldn't the frame for constructors have first the values, then the hole, and then the unevaluated terms?

Reply:
Yes, I swapped them at some point and missed this one, thanks!